src/Action/ResetAction.php

```diff
@@ -13,28 +13,23 @@
 
 namespace Sonata\UserBundle\Action;
 
-use FOS\UserBundle\Form\Factory\FactoryInterface;
-use FOS\UserBundle\Model\UserManagerInterface;
-use FOS\UserBundle\Security\LoginManagerInterface;
-use Psr\Log\LoggerAwareTrait;
-use Psr\Log\NullLogger;
 use Sonata\AdminBundle\Admin\Pool;
 use Sonata\AdminBundle\Templating\TemplateRegistryInterface;
+use Sonata\UserBundle\Form\Type\ResettingFormType;
+use Sonata\UserBundle\Model\UserManagerInterface;
+use Symfony\Component\Form\FormFactoryInterface;
```

Comment:
We should remove reset password action in favor of [symfonycasts/reset-password-bundle](https://github.com/SymfonyCasts/reset-password-bundle)
It is smaller dedicated for reset password (data are stored in `reset_password_request`) so `User` will be cleaner.

Simmilar to this we should remove email verification action in favor of [symfonycasts/verify-email-bundle](https://github.com/SymfonyCasts/verify-email-bundle)

This will allow to drop a lot of things from `FOSUserBundle`. 

IMO From `FOSUserBundle` we should move only `Group - User(ROLE)` structure.

Reply:
There are several things in your comment that I dont agree.

1. The reset password request is a feature already provided by this bundle. You are pointing to a file that is already present on this bundle and I am only changing some uses. There is no need to replace it with the bundle you mention in order to remove FOSUser (It can be done later if needed)
2. We are not doing any email verification in this bundle. Not previously with FOSUser installed, and not now that we remove it, so I dont understand what you mean. I dont want to implement new features in order to remove a bundle
3. Even FOSUser removed the group features, it is a feature that only brings more queries to the database in order to retrieve the permissions. Is not something we should encourage to use. The only reason I keep it in this Pr is because this is only about the minimal steps to remove FOSUser. In nexts Prs I will probably recommend deprecating it too. But thats a discussion for later.